src/rpc/blockchain.cpp

```diff
@@ -331,18 +331,18 @@ UniValue getdifficulty(const UniValue& params, bool fHelp)
 
 std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,             (numeric) transaction size in bytes\n"
+    return "    \"size\" : n,             (numeric) virtual transaction size. Differs from size for witness transactions since witness size is discounted. BIP 141.\n"
```

Comment:
This doesnt elucidate what this is to me -  maybe the second sentence should be like "Differs from size for witness transactions as this does not include the witness data" or something like that (I dont actually know what its supposed to be).